command/agent.go

```diff
@@ -343,6 +340,20 @@ func (c *AgentCommand) Run(args []string) int {
 			return 1
 		}
 
+		sinkConfig := &sink.SinkConfig{
```

Comment:
Could we only set this up if use auto auth is enabled?

Reply:
Done.